service/qos/qos_configuration_change_subscriber.hh

```diff
@@ -16,14 +16,19 @@ namespace qos {
     struct service_level_info {
         sstring name;
     };
+
     class qos_configuration_change_subscriber {
     public:
         /** This callback is going to be called just before the service level is available **/
         virtual future<> on_before_service_level_add(service_level_options slo, service_level_info sl_info) = 0;
-        /** This callback is going to be called just after the service level is removed **/
+        /** This callback is going to be called just after the service level is available **/
+        virtual future<> on_after_service_level_add(service_level_options slo, service_level_info sl_info) = 0;
```

Comment:
noexcept?

Reply:
Or rather, a statement about how this may not fail, since an exception can be returned in the future too.